pkg/pulumiyaml/run_test.go

```diff
@@ -409,10 +429,12 @@ outputs:
 func TestAssetOrArchive(t *testing.T) {
 	t.Parallel()
 
-	const text = `name: test-yaml
+	file := createTestFile(t)
```

Comment:
Bazel symlinks all files to its own directory structure, to be able to know the exact inputs for each build.  In this particular case, if we have a symlink pulumi-yaml tries to follow it, and then realizes that it's outside of the directory we're working in, and errors out.  We're generating a test file here instead, which seems no worse than relying on the readme here.